.github/workflows/bundler-integrations.yml

```diff
@@ -0,0 +1,42 @@
+name: Bundler integrations
+
+on:
+  workflow_call:
+  workflow_dispatch:
+
+jobs:
+  test-tree-shaking:
+    name: Test tree shaking
+    runs-on: ubuntu-latest
+
+    env:
+      PUPPETEER_SKIP_DOWNLOAD: true
+
+    strategy:
+      fail-fast: false
+
+      matrix:
+        bundler:
+          - rollup
+          - webpack
+          - vite
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4.1.4
+
+      - name: Restore dependencies
+        uses: ./.github/workflows/actions/install-node
+
+      - name: Build GOV.UK Frontend
+        uses: ./.github/workflows/actions/build
+
+      - name: Build with bundler
+        run: npm run ${{matrix.bundler}} -w @govuk-frontend/bundler-integrations
+
+      # Check output for modules that should not be included
+      - name: Check output
+        working-directory: ./.github/workflows/bundler-integrations
+        run: |
+          ! grep "Accordion" dist/${{matrix.bundler}}/default.js
+          grep "Accordion" dist/${{matrix.bundler}}/initAll.js
```

Comment:
**suggestion** The [output from the grep](https://github.com/alphagov/govuk-frontend/actions/runs/8921612225/job/24502174396?pr=4961#step:6:8) isn't particularly useful so we could consider silencing it with `--quiet` (or `-q`)